docs/references/configuration.md

```diff
@@ -247,3 +246,22 @@ Note that the controller does not get deployed when initiating `create` with the
 In that case, `controller_image` must still be set but is ignored.
 
 Change this (compared to the default value) only when you know what you're doing :-).
+
+### `log_retention_days` and `metrics_retention_days`
+
+Defines data retention in terms of the number of desired days (for each data type).
+
+*Default:* 7 days.
+
+*Value type:* number \(integer\)
+
+```yaml
+log_retention_days: 90
+metrics_retention_days: 365
+```
+
+Notes:
+
+* These options currently cannot be specified on a per-tenant basis.
+* These options currently cannot be changed after initial cluster creation.
```

Comment:
> These options currently cannot be changed after initial cluster creation.

Fair enough! But this might make it sound like the other parameters can be changed after initial cluster creation :) Most of them cannot. We do not have a config mutation concept, we never paid attention to this. If any one of the parameters can currently be changed upon "upgrade" then this work so to say as of being lucky, not by design.

One way to approach this is to qualify individual parameters one-by-one in manual testing, and then add a note in the lines of "this parameter can be changed ... ".

Reply:
Actually, none of the config file parameters can be changed during the upgrade.  So I'll note that and merge.

Reply:
> Wait, is this correct?

I think I tried to answer that with my comment above. Let me know which part needs clarification.